chart/etcd-backup-restore/values.yaml

```diff
@@ -31,6 +31,9 @@ servicePorts:
   server: 2380
   backupRestore: 8080
 
+etcdAuth:
```

Comment:
This block can be extended someday to include client cert and key for TLS client auth.